conda/packages/nv_ingest/meta.yaml

```diff
@@ -66,6 +66,9 @@ requirements:
     - unstructured-client>=0.25.9
     - uvicorn
     - wand>=0.6.10
+  pip:
+    - ffmpeg>=1.1
```

Comment:
Similar comment as above, but it looks like `meta.yaml` is missing `ffmpeg-python>=0.2.0`.